### ISSUE TITLE
Draining tracts in skin

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9996]
**Preferred term label:**
Draining tracts in skin

**Synonyms**
Tracts under skin draining associated abscess, discharging sinuses, fistulae

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Hidradenitis suppurativa
Mycetoma

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000095
- label: Draining sinus tract in skin
- parents: [Localized skin lesion (HP:0011355)](https://hpo.jax.org/app/browse/term/HP:0011355)
- pmid: PMID:34145466
- def: A sinus tract is an abnormal channel that originates from the skin or a mucous surface to a deep-seated focus of suppuration.
- comment: A soft tissue ulcer can show a sinus tract leading to a deeper soft tissue abscess and may occasionally extend to a joint. Chronic osteomyelitis can be associated with a sinus tract and cloaca (opening or rupture of bony cortex overlying an area of osteomyelitis) draining granulation tissue, pus, or necrotic debris from the bone to the skin.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9996
- synonyms: Tract under skin draining associated abscess(NONE)|Discharging sinus(NONE)|Draining tract in skin(NONE)
